lib/fob.ex

```diff
@@ -39,15 +39,25 @@ defmodule Fob do
     limit(queryable, ^page_size)
   end
 
-  defp apply_keyset_comparison(
+  defp route_keyset_comparison(routeable_page_break, accumulator) do
```

Comment:
Didn't notice until later :neckbeard: feedback, minimal `acc` norm in Elixir, so one of the times would accept arguments for convention > clarity, given the `acc` conceptual importance imo.